src/analysis/normed/order/lattice.lean

```diff
@@ -31,30 +31,42 @@ normed, lattice, ordered, group
 -/
 
 /-!
-### Normed lattice orderd groups
+### Normed lattice ordered groups
 
 Motivated by the theory of Banach Lattices, this section introduces normed lattice ordered groups.
 -/
 
 local notation (name := abs) `|`a`|` := abs a
 
+section solid_norm
+
+class has_solid_norm (α : Type*) [normed_add_comm_group α] [lattice α]  : Prop :=
+(solid :  ∀ ⦃x y : α⦄, |x| ≤ |y| → ‖x‖ ≤ ‖y‖)
+
+variables {α : Type*} [normed_add_comm_group α] [lattice α]
+
+lemma solid [has_solid_norm α] {a b : α} (h : |a| ≤ |b|): ‖a‖ ≤ ‖b‖ := has_solid_norm.solid h
```

Comment:
This seems a bit too generic to not be in a namespace, can't we just `alias has_solid_norm.solid ← norm_le_norm_of_abs_le_abs`

Reply:
That's a good idea but then lint complains 
```
norm_le_norm_of_abs_le_abs - is a def, should be a lemma/theorem
```
and I don't know how to fix that

Reply:
This is the alias / structure command being broken; you can just edit the name of this lemma jnstead